components/events/TheEventsHeader.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <ThePageHeader class="the-learn-header">
+    Join events from the worlds
```

Comment:
ooh, I also think this should be `world‘s`, yeah?

Reply:
I don't know, I just take the text from the Figma file 😅

Reply:
(checked with Russell, and it should read `world‘s` :pray: